staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/fuzzer/fuzzer.go

```diff
@@ -61,6 +61,18 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 					{Name: "Age", Type: "date", Description: swaggerMetadataDescriptions["creationTimestamp"], JSONPath: ".metadata.creationTimestamp"},
 				}
 			}
+			for _, v := range obj.Versions {
+				if len(v.AdditionalPrinterColumns) == 0 {
```

Comment:
can we have better fuzzing here? AdditionalPrinterColumns is optional and it should be only randomly set.

Reply:
I suppose the fuzzing for AdditionalPrinterColumns was to match the defaulter. We default the top level AdditionalPrinterColumns if [`len(obj.AdditionalPrinterColumns) == 0`](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/defaults.go#L69-L73). 

Removed the per-version fuzzing since we don't do defaulting for per-version fields now.